pkg/svc/application.go

```diff
@@ -0,0 +1,120 @@
+package svc
+
+import (
+	"context"
+	"github.com/pkg/errors"
+	"log"
+
+	"github.com/fuseml/fuseml-core/gen/application"
+	"github.com/fuseml/fuseml-core/pkg/domain"
+	"github.com/fuseml/fuseml-core/pkg/kubernetes"
+)
+
+func appRestToDomain(ra *application.Application) (a *domain.Application, err error) {
```

Comment:
Ooops, I just realized that what I did with codesets was in the same `svc` package, and the function names I used there were too generic (`restToDomain` and `domainToRest`). Sorry about that.

Reply:
yep, and I was wondering why does it work for you when you already added it for codesets and runnables ... and I realized it's because you do not rebase! :-)

Reply:
yes, I need to do that more often :-)